cfgov/unprocessed/css/enhancements/layout.less

```diff
@@ -311,6 +311,81 @@
 }
 
 /* topdoc
+  name: Adds block background that bleeds off page 
+  family: cf-layout
+  notes:
+    - ".block__bg-extend-right requires that overflow-x be set to 
+       hidden on a parent element."
+  tags:
+    - cf-layout
+*/
+
+.block {
```

Comment:
I kept these as block modifiers, but scoped them under the Ask CFPB page class for now.